application/service/services.go

```diff
@@ -79,6 +79,12 @@ type NotificationService interface {
 	SendMessagesAsync(ctx context.Context, messages []notification.Message) error
 }
 
+type WITService interface {
+	UpdateWITUser(ctx context.Context, updatePayload *app.UpdateUsersPayload, identityID string) error
+	CreateWITUser(ctx context.Context, identity *account.Identity, identityID string) error
+	GetSpaceNameAndOwnedBy(ctx context.Context, spaceID string) (name, ownedBy string, e error)
```

Comment:
Very soon we might need more info from the space service - let's return a struct here. 
```
type Space struct {
	ID              uuid.UUID
	Name            string
	Description     string
	OwnerID         uuid.UUID 
}
```